packages/eslint-plugin/src/rules/member-delimiter-style.ts

```diff
@@ -21,14 +28,26 @@ type Config = BaseOptions & {
     typeLiteral?: BaseOptions;
     interface?: BaseOptions;
   };
-  multilineDetection?: 'brackets' | 'last-member';
```

Comment:
why have you removed this option entirely?

Reply:
Fixed here: e5353a9ea7bdff443335c057e61f75de893ec901